README.md

```diff
@@ -8,7 +8,11 @@ Idea is based on Stalin Sort ![ss](https://i.redd.it/x9triplll1v11.jpg)
 
 It's alike binary search but any checking element which is not target one is eliminated.
 
-Complexity is~ O(log n) on first run however after~ n/2 runs Complxity will be O(1) guaranteed.
+The worst running complexity is ~N. The best running complexity is ~1.  
```

Comment:
was targeting sorted arrays obviously, working on unsorted arrays just not to work correctly on any data, it makes code little bit more complicated but there is it.

Reply:
There were tests targeting the unsorted data (at least as far as google translated code seemed to mean to me).
Anyways, there was another pr concerning this thing